src/bindings/python/src/openvino/runtime/opset14/ops.py

```diff
@@ -4,30 +4,48 @@
 
 """Factory functions for ops added to openvino opset14."""
 from functools import partial
-from typing import Literal, Optional, Union
-import logging
-
-import numpy as np
-
-log = logging.getLogger(__name__)
 
 from openvino.runtime import Node
 from openvino.runtime.opset_utils import _get_node_factory
 from openvino.runtime.utils.decorators import nameable_op
-from openvino.runtime.utils.types import (
-    NodeInput,
-    as_node,
-    as_nodes
-)
+from openvino.runtime.utils.types import NodeInput, as_nodes
 
 _get_node_factory_opset14 = partial(_get_node_factory, "opset14")
 
 
 # -------------------------------------------- ops ------------------------------------------------
+@nameable_op
+def convert_promote_types(
+    left_node: NodeInput,
+    right_node: NodeInput,
+    promote_unsafe: bool = False,
+    pytorch_scalar_promotion: bool = False,
+    u64_integer_promotion_target: str = "f32",
```

Comment:
```suggestion
    u64_integer_promotion_target: Union[str, ov.Type] = "f32",
```
Would it be possible to pass OV types as well? (*)

Reply:
Thanks, done, ov Type can be passed. Modified tests to use both str and Type